.cloudgov/vars/pages-production.yml

```diff
@@ -0,0 +1,8 @@
+env: production
+env_postfix: -production
```

Comment:
Conventionally, I think we've been keeping the `env_postfix` value as a blank string `''`. Ie https://github.com/18F/federalist-builder/blob/staging/.cloudgov/vars/production.yml#L2

Reply:
Oops, fixed!